FeedAPIChallenge/Feed API/RemoteFeedLoader.swift

```diff
@@ -19,6 +19,14 @@ public final class RemoteFeedLoader: FeedLoader {
 	}
 
 	public func load(completion: @escaping (FeedLoader.Result) -> Void) {
-		fatalError("Must be implemented")
+		client.get(from: url) { [weak self] response in
+			guard self != nil else { return }
+			switch response {
+			case let .success((data, httpResponse)):
+				completion(FeedImageMapper.resolves(data, with: httpResponse))
+			case .failure:
+				completion(.failure(RemoteFeedLoader.Error.connectivity))
```

Comment:
We can shorten the error declaration since this portion of code is in the same scope as `RemoteFeedLoader`:
```suggestion
				completion(.failure(Error.connectivity))
```

Reply:
Done